subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/WithAndroidDeprecations.groovy

```diff
@@ -25,6 +25,7 @@ trait WithAndroidDeprecations implements WithReportDeprecations {
     private static final VersionNumber AGP_VERSION_WITH_FIXED_NEW_WORKERS_API = VersionNumber.parse('4.2')
     private static final VersionNumber AGP_VERSION_WITHOUT_CONVENTION_USAGES = VersionNumber.parse('7.4')
     private static final VersionNumber AGP_VERSION_WITHOUT_CONFIG_UTIL = VersionNumber.parse('8.0.0-rc01')
+    private static final VersionNumber AGP_VERSION_WITHOUT_CLIENT_MODULE = VersionNumber.parse('8.2.0-alpha06')
```

Comment:
❓  Can we really start nagging if there is no released AGP version that will not trigger the deprecation?

Reply:
Probably not. The change in Android to remove client modules was trivial. Perhaps we can get android to backport it to 8.1 which is in beta now?

I can comment on the AGP issue, though idk how stable 8.1 is now given it's in beta.

Reply:
We decided to continue with the deprecation in terms of adding `@Deprecated` annotations and adding the upgrade guide section. However, we removed the nagging and wrote up https://github.com/gradle/gradle/issues/25403 issue to revisit it when AGP 8.2.0 is released